### PR TITLE
[ADD] website_legal_page addon

### PR DESCRIPTION
**Pull request by antespi**
Changes in website_legal_page.Added legal texts
[FIX] layout and text fixes. Added credits to readme
Update changes
Translates to Spanish
Last review

### CODE REVIEW

File: website_legal_page/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: AGPLv3 License
+
+Website legal page
+==================
+
+This module was written to provides common legal pages needed in any website:
```
Comment (pedrobaeza) [inline website_legal_page/README.rst]: s/to provides/to provide

File: website_legal_page/README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* **Terms of use**: Which services website offers and legal restrictions
+applied to them
+* **Privacy policy**: Company privacy policy applied to user data collected
+through the website
+
+
+Usage
+=====
+
+Website editors can change easily any text of these pages using website builder
```
Comment (pedrobaeza) [inline website_legal_page/README.rst]: s/editors/editor

File: website_legal_page/README.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+Disclaimer
```
Comment (pedrobaeza) [inline website_legal_page/README.rst:25]: Call it `Known issues / Roadmap`

File: website_legal_page/README.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Authors, contributors, maintainer and any of its employees, contractors, or
+attorneys who participated in development of this addon **expressly disclaim any
+warranty**: they are not creating or entering into any Attorney-Client
+relationship by providing information to you.
+
+
+Credits
+=======
+
+Templates are based on legal templates publicy provided by [termsfeed.com](https://termsfeed.com):
```
Comment (pedrobaeza) [inline website_legal_page/README.rst:63]: Put a section called `Templates` for homogeneity

File: website_legal_page/__init__.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
```
Comment (pedrobaeza) [inline website_legal_page/__init__.py]: Don't put header if there is no content

File: website_legal_page/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (pedrobaeza) [inline website_legal_page/__openerp__.py]: Remove this line

File: website_legal_page/i18n/es.po [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+#: view:website:website.terms
+msgid "(\"us\", \"we\", or \"our\")."
+msgstr "(\"nos\", \"nosotros\", or \"nuestro\")."
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "(the \"Site\"). This page informs you of our policies regarding the collection, use and disclosure\n"
+"                  of Personal Information we receive from users of the Site."
+msgstr "la \"pagina\". Esta página le informa de nuestras políticas en cuanto"
+       " a la colección, uso y revelación de la Información personal que"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: This indentation is not correct

File: website_legal_page/i18n/es.po [hunk 8]
```diff
@@ -0,0 +1,338 @@
...
+"                  without regard to its conflict of law provisions."
+msgstr "sin tener en cuenta su conflicto de disposiciones legales."
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "All provisions of the Terms which by their nature should survive termination shall survive\n"
+"                  termination, including, without limitation, ownership provisions, warranty disclaimers, indemnity\n"
+"                  and limitations of liability."
+msgstr "Todas las disposiciones de los términos que por su naturaleza deberían"
+       " sobrevivir a la terminación sobrevivirán a la terminación, incluyendo"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: This indentation is not correct

File: website_legal_page/i18n/es.po [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+       " las disposiciones de propiedad, renuncias de garantía, indemnización"
+       " y limitaciones de responsabilidad."
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "By accessing or using the Service you agree to be bound by these Terms. If you disagree with\n"
+"                  any part of the terms then you may not access the Service."
+msgstr "Al acceder o utilizar el servicio usted acepta estar obligado por estos"
+       " términos. Si está en desacuerdo con cualquier parte de los términos"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: This indentation is not correct

File: website_legal_page/i18n/es.po [hunk 10]
```diff
@@ -0,0 +1,338 @@
...
+#: view:website:website.privacy
+msgid "Cookies"
+msgstr "Cookies"
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "Cookies are files with small amount of data, which may include an anonymous unique identifier.\n"
+"                  Cookies are sent to your browser from a web site and stored on your computer's hard drive."
+msgstr "Las cookies son archivos con una pequeña cantidad de datos que pueden"
+       " incluir un identificador único anónimo. Las cookies son enviadas a su"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: This indentation is not correct

File: website_legal_page/i18n/es.po [hunk 11]
```diff
@@ -0,0 +1,338 @@
...
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "Governing Law"
+msgstr "Jurisdicción"
+
+#. module: website_legal_page
+#: view:website:website.privacy
+#: view:website:website.terms
+msgid "If you have any questions about these Terms, please contact us."
+msgstr "Si tiene alguna pregrunta acerca de estos Términos, por favor contacte con nosotros."
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: s/pregrunta/pregunta

File: website_legal_page/i18n/es.po [hunk 12]
```diff
@@ -0,0 +1,338 @@
...
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "Information Collection And Use"
+msgstr "Recopilación y uso"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Información acerca de la recopilación y uso

File: website_legal_page/i18n/es.po [hunk 13]
```diff
@@ -0,0 +1,338 @@
...
+msgstr "Las cookies son archivos con una pequeña cantidad de datos que pueden"
+       " incluir un identificador único anónimo. Las cookies son enviadas a su"
+       " navegador por un sitio web y almacenadas en el disco duro de su ordenador."
+
+#. module: website_legal_page
+#: view:website:website.legal
+#: view:website:website.privacy
+#: view:website:website.terms
+msgid "Email address:"
+msgstr "Dirección de email:"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: s/email/correo electrónico

File: website_legal_page/i18n/es.po [hunk 14]
```diff
@@ -0,0 +1,338 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "(\"us\", \"we\", or \"our\") operates"
+msgstr "(\nos\",\"nosotros"\"o \"nuestro\" opera"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: close parenthesis

File: website_legal_page/i18n/es.po [hunk 15]
```diff
@@ -0,0 +1,338 @@
...
+msgstr "Aviso Legal"
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "Like many site operators, we collect information that your browser sends whenever you visit our\n"
+"                  Site (\"Log Data\"). This Log Data may include information such as your computer's Internet\n"
+"                  Protocol (\"IP\") address, browser type, browser version, the pages of our Site that you visit, the\n"
+"                  time and date of your visit, the time spent on those pages and other statistics."
+msgstr "Como muchos otros operadores, coleccionamos la información que su"
+       " navegador envía siempre que visita nuestro Sitio web (\"Datos del Usuario\")."
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 16]
```diff
@@ -0,0 +1,338 @@
...
+       " navegador, las páginas de nuestro Sitio web que visita, el tiempo y"
+       " la fecha de su visita, el tiempo pasado en aquellas páginas y otras estadísticas."
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "Like many sites, we use \"cookies\" to collect information. You can instruct your browser to refuse\n"
+"                  all cookies or to indicate when a cookie is being sent. However, if you do not accept cookies, you\n"
+"                  may not be able to use some portions of our Site."
+msgstr "Como muchos sitios, usamos cookies para coleccionar la información."
+       " Puede configurar su navegador para rechazar todas las cookies o"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 17]
```diff
@@ -0,0 +1,338 @@
...
+#: view:website:website.terms
+msgid "Our Service may contain links to third­party web sites or services that are not owned or controlled\n"
+"                  by ​us. We has no control over, and assumes no responsibility for, the\n"
+"                  content, privacy policies, or practices of any third party web sites or services. You further\n"
+"                  acknowledge and agree that ​we shall not be responsible or liable,\n"
+"                  directly or indirectly, for any damage or loss caused or alleged to be caused by or in connection\n"
+"                  with use of or reliance on any such content, goods or services available on or through any such\n"
+"                  web sites or services."
+msgstr "Nuestro servicio puede contener enlaces a sitios web de terceros o"
+       " servicios que no son propiedad o controlados por nosotros. No tenemos"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 18]
```diff
@@ -0,0 +1,338 @@
...
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "Our failure to enforce any right or provision of these Terms will not be considered a waiver of\n"
+"                  those rights. If any provision of these Terms is held to be invalid or unenforceable by a court, the\n"
+"                  remaining provisions of these Terms will remain in effect. These Terms constitute the entire\n"
+"                  agreement between us regarding our Service, and supersede and replace any prior agreements\n"
+"                  we might have between us regarding the Service."
+msgstr "Nuestra incapacidad para hacer cumplir cualquier derecho o disposición"
+       " de estos términos no se considerará una renuncia a dichos derechos."
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 19]
```diff
@@ -0,0 +1,338 @@
...
+#: view:website:website.legal
+#: view:website:website.privacy
+#: view:website:website.terms
+msgid "Phone number:"
+msgstr "Número de teléfono:"
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "Please read these Terms of Use (\"Terms\", \"Terms of Use\") carefully before using the"
+msgstr "Por favor lea antetamente estos Términos de Uso antes de usar"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: s/antetamente/atentamente

Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: s/antes de usar/antes de continuar

File: website_legal_page/i18n/es.po [hunk 20]
```diff
@@ -0,0 +1,338 @@
...
+msgstr "El propietario de esta web es"
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "The security of your Personal Information is important to us, but remember that no method of\n"
+"                  transmission over the Internet, or method of electronic storage, is 100% secure. While we strive to\n"
+"                  use commercially acceptable means to protect your Personal Information, we cannot guarantee\n"
+"                  its absolute security."
+msgstr "La seguridad de su Información personal es importante para nosotros,"
+       " pero recuerde que ningún método de la transmisión sobre Internet o"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 21]
```diff
@@ -0,0 +1,338 @@
...
+       " método de almacenaje electrónico, es 100% seguro. Si bien nos"
+       " esforzamos por utilizar medios comercialmente aceptables para"
+       " proteger su información personal, no podemos garantizar su seguridad."
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "These Terms shall be governed and construed in accordance with the laws of ​"
+msgstr "Estos Términos se regirán e interpretarán de conformidad con las leyes de"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: space at the end

File: website_legal_page/i18n/es.po [hunk 22]
```diff
@@ -0,0 +1,338 @@
...
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "We may terminate or suspend access to our Service immediately, without prior notice or liability,\n"
+"                  for any reason whatsoever, including without limitation if you breach the Terms."
+msgstr "Podemos cancelar o suspender el acceso a nuestro servicio"
+       " inmediatamente sin previo aviso o responsabilidad por cualquier"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 23]
```diff
@@ -0,0 +1,338 @@
...
+       " razón sin limitación en caso de incumplimiento de los términos."
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "We may update this Privacy Policy from time to time. We will notify\n"
+"                  you of any changes by posting the new Privacy Policy on the Site. You are advised to review this\n"
+"                  Privacy Policy periodically for any changes."
+msgstr "Podemos actualizar cambios en esta Política de privacidad de vez"
+       " en cuando. Le notificaremos cualquier cambio fijando la nueva"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 24]
```diff
@@ -0,0 +1,338 @@
...
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "We reserve the right, at our sole discretion, to modify or replace these Terms at any time. If a\n"
+"                  revision is material we will try to provide at least ​30 days notice prior to any new\n"
+"                  terms taking effect. What constitutes a material change will be determined at our sole discretion.\n"
+"                  By continuing to access or use our Service after those revisions become effective, you agree to\n"
+"                  be bound by the revised terms. If you do not agree to the new terms, please stop using the\n"
+"                  Service."
+msgstr "Reservamos el derecho, a nuestra propia discreción, de modificar o"
+       " sustituir estos Términos en cualquier momento. Si una revisión es"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 25]
```diff
@@ -0,0 +1,338 @@
...
+       " aquellas revisiones entren en vigor, consiente en ser ligado por los"
+       " términos revisados. Si no está de acuerdo con los nuevos términos,"
+       " por favor deje de usar el Servicio"
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "We strongly advise you to read the terms and conditions and privacy policies of any third­party\n"
+"                  web sites or services that you visit."
+msgstr "Aconsejamos que lea los términos y condiciones y políticas de"
+       " privacidad de los sitios web de terceros o servicios que usted visite."
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 26]
```diff
@@ -0,0 +1,338 @@
...
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "We use your Personal Information only for providing and improving the Site. By using the Site,\n"
+"                  you agree to the collection and use of information in accordance with this policy."
+msgstr "Usamos su Informacion Personal solamente para proporcionar y mejorar"
+       " el sitio. Al utilizar el Sitio, usted acepta la recopilación y uso de"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 27]
```diff
@@ -0,0 +1,338 @@
...
+       " información de acuerdo con esta política."
+
+#. module: website_legal_page
+#: view:website:website.privacy
+msgid "While using our Site, we may ask you to provide us with certain personally identifiable information\n"
+"                  that can be used to contact or identify you. Personally identifiable information may include, but is\n"
+"                  not limited to your name (\"Personal Information\")."
+msgstr "Durante el uso de nuestro sitio podremos pedirle que nos proporcione"
+       " cierta información que puede ser usada para contactar o identificarle."
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/i18n/es.po [hunk 28]
```diff
@@ -0,0 +1,338 @@
...
+#: view:website:website.legal
+msgid "You can contact us at:"
+msgstr "Puede contactar con nosotros en:"
+
+#. module: website_legal_page
+#: view:website:website.terms
+msgid "Your access to and use of the Service is conditioned on your acceptance of and compliance with\n"
+"                  these Terms. These Terms apply to all visitors, users and others who access or use the Service."
+msgstr "El acceso y uso del servicio está condicionado a la aceptación y"
+       " cumplimiento de estos términos. Estos términos se aplican a todos los"
```
Comment (pedrobaeza) [inline website_legal_page/i18n/es.po]: Identation not correct

File: website_legal_page/views/website_legal.xml [hunk 29]
```diff
@@ -0,0 +1,89 @@
...
+                  <t t-if="res_company.country_id">
+                    <span t-field="res_company.country_id.name"/>
+                  </t>
+                </t>
+              </p>
+            </div>
+            <div>
+              <t t-if="res_company.vat">
+                <p>
+                  TAX ID: <span t-field="res_company.vat"/>
```
Comment (pedrobaeza) [inline website_legal_page/views/website_legal.xml]: VAT number is more correct

**Comment by pedrobaeza** [comment]
For the icon, you can combine the website icon with the scale you have put. Add credits for the scale and include the SVG source if you have it for possible modifications

**Comment by antespi** [comment]
@pedrobaeza I got scale image from: https://www.google.es/search?q=legal&espv=2&biw=1242&bih=575&source=lnms&tbm=isch&sa=X&ei=3KaIVcC0HIvaUav8g3g&ved=0CAYQ_AUoAQ#tbm=isch&tbs=rimg%3ACY43We5mEX-pIjgqHKHGZ8JYPrhb0I8uroeQB2inpJ_19xiW2OKhUNWkDH0DMcdwHJWs9fl5Ih8GZdVGewHm5DkvLBCoSCSococZnwlg-EVhE73vXQh95KhIJuFvQjy6uh5ARrnSsLPlvu8kqEgkHaKekn_13GJRFrmudKMHB8qyoSCbY4qFQ1aQMfEajyhz4ucFZMKhIJQMxx3Aclaz0RFOiMfQGXoxsqEgl-XkiHwZl1URFoI8rwB7IfvSoSCZ7AebkOS8sEEfSzQsu8rkMy&q=legal&imgrc=jjdZ7mYRf6nPYM%253A%3BqFR_TmfzdXrEkM%3Bhttp%253A%252F%252Fwww.kaybelsolutions.com%252Fwp-content%252Fuploads%252F2012%252F11%252Flegal.jpg%3Bhttp%253A%252F%252Fwww.kaybelsolutions.com%252Faviso-legal%252F%3B400%3B300

I have no SVG version. I didn't combine with website icon because it is a snapshot of a website and didn't match very well with scale

**Comment by pedrobaeza** [comment]
You need to check then if there is any restriction in the use of the icon. You can use any of the free SVG images in here: https://openclipart.org/search/?query=scale

**Comment by antespi** [comment]
@pedrobaeza all suggestion changed, thanks for review

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**